app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -22,8 +23,18 @@ angular.module("openshiftConsole")
         }, {
           value: "Pi",
           label: "PiB"
-        }];
+          }];
         scope.claim.selectedLabels = [];
+        DataService.get({
+          group: 'storage.k8s.io',
+          resource: 'storageclasses'
+        }, null, {}).then(
```

Comment:
Unless the current $scope here includes the project / projectName and then it would try to namespace scope the request which isnt what you want.  If thats the case you may just have to pass an empty map here `{}`